src/components/Card.js

```diff
@@ -47,11 +60,23 @@ export default class Card {
     this._cardCaption = this._cardElement.querySelector("#card-title-id");
     this._cardImage = this._cardElement.querySelector("#card__image-modal");
     this._cardImage.src = this._link;
+    this._cardLikeButton = this._cardElement.querySelector("#card-like-button");
     this._cardImage.alt = this._name;
     this._cardCaption.textContent = this._name;
-    //get the card view
-    //set event listeners
+    this._updateLikesView();
```

Comment:
`_updateLikesView` 

* When I like items, there are errors https://snipboard.io/Wf9iKg.jpg

You should remove the `_` (**underscore**) from the method, because you use it in `index.js`. So, it should be **public**


```js
this.updateLikesView();

```